recipe/meta.yaml

```diff
@@ -50,8 +41,10 @@ requirements:
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}
     - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    - cf-nvidia-tools                       # [linux]
```

Comment:
Added `cf-nvidia-tools` and switched to it for our GLIBC check

Think I got the related bits replaced. Though please let me know if I've missed something

Reply:
LGTM

Reply:
```suggestion
    - cf-nvidia-tools 1                       # [linux]
```
Also, maybe we want to be conservative and pin to the major version to protect against breaking changes?

Reply:
Are there particular breaking changes you have in mind?

Reply:
Not currently, but I've promised to use actual semantic versioning for this package. I suspect there may be breaks once we start adding more features to cf-nvidia-tools and the complexity increases.

Reply:
Ok let's make sure this goes in the migrator and other used places